qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -1030,11 +1030,14 @@ def test_create_qiime_mapping_file(self):
         obs_fp = pt.create_qiime_mapping_file()
         exp_fp = join(fp, '1_prep_1_qiime_19700101-000000.txt')
 
-        obs = pd.read_csv(obs_fp, sep='\t', infer_datetime_format=True,
-                          parse_dates=True, index_col=False, comment='\t')
+        obs = pd.read_csv(obs_fp, sep='\t', infer_datetime_format=False,
+                          parse_dates=False, index_col=False, comment='\t',
+                          na_values=qdb.metadata_template.constants.NA_VALUES,
+                          true_values=qdb.metadata_template.constants.TRUE_VALUES,
```

Comment:
This parsing is still inferring boolean values (true_values/false_values) and changing some of the user provided values to `Null` which will later be changed to 'Unknown' (na_values). I think a better goal is to make sure that we don't 'infer' anything and we are able to write the exact same thing that we are reading.